app/controllers/spina/admin/images_controller.rb

```diff
@@ -35,8 +35,15 @@ def create
           image
         end.compact
 
-        if params[:modal]
+        if params[:origin] == 'media-picker'
           redirect_to spina.admin_media_picker_path(media_folder_id: image_params[:media_folder_id])
+        elsif params[:origin] == 'auto-file-upload'
+          render turbo_stream: turbo_stream.update(
+            'auto-file-upload-images',
+            Spina::Forms::EditorInsertImagesMetaComponent
+              .new(images: @images, trix_target_id: params[:trix_target_id])
+              .render_in(view_context)
+          )
         else
           render turbo_stream: turbo_stream.prepend("images", partial: "image", collection: @images)
```

Comment:
Makes use of existing ImagesController to perform the same processing logic, just rerouting the output depending on where the upload originated from:
* Media Picker Page
* Media Libary Page
* Auto File Upload Component